src/pages/o/[orgId]/surveys/[surveyId]/index.tsx

```diff
@@ -221,6 +226,11 @@ const Page: FC<PageProps> = ({ orgId, status, survey }) => {
               <>
                 {element.question.response_type === 'text' && (
                   <TextQuestion
+                    defaultValue={
```

Comment:
I like this solution and agree with you that keeping state management out of this is the way to go, for as long as we can. 😊